### PR TITLE
console blueprint for navigator ft

### PR DESCRIPTION
**Pull request by yamini**
New NavFT console blueprint card (and a couple of small updates):
- Created a new card in gretel.json
- Added sample dataset and generated a preview
- Added hero image for details page
- Created details page
- Removed the relational synthetics card
- Updated actgan card copy
- Rearranged tv2 and tuner cards and updated copy

### CODE REVIEW

File: use_cases/details/navigator-ft.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+![Generate synthetic tabular, text and time series data](https://blueprints.gretel.cloud/use_cases/images/navigator-ft-hero.png "Generate synthetic tabular, text and time series data")
+
+We are excited to announce the public preview of **Navigator fine-tuning**, the latest advancement in our suite of synthetic data solutions. This new feature builds upon the recent general availability of [Gretel Navigator](https://console.gretel.ai/navigator), enabling you to generate data not only from a prompt, but also from fine-tuning the underlying model on your domain-specific real-world datasets to generate the highest quality synthetic data.
```
Comment (thesteady) [inline use_cases/details/navigator-ft.md:3]: > We are excited to announce the public preview of **Navigator fine-tuning**

Should "fine-tuning" be capitalized? Also on line 5.

**Review by thesteady — COMMENTED** [review]
To get the tests to pass, you need to add `"navigator_ft",` to the list of accepted modelTypes in the `test_use_cases.py` test file.

File: use_cases/details/navigator-ft.md (hunk 1 above)
Comment (yamini) [reply use_cases/details/navigator-ft.md:3]: @sami-t ? I copied that from the blog post.

Comment (sami-t) [reply use_cases/details/navigator-ft.md:3]: Let's go with Navigator Fine Tuning (capitalized and without the dash). The blog post draft is already updated. Thanks!

**Review by thesteady — APPROVED** [review]

**Pull request merged**